app/res/layout/question_widget_text_compact.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<TextView xmlns:android="http://schemas.android.com/apk/res/android"
+          android:layout_width="0dp"
+          android:layout_weight="1"
+          android:layout_height="match_parent"
+          android:ellipsize="end"
+          android:maxLines="1"
```

Comment:
Is this actually what we want? I wasn't imagining that a requirement for this would be limiting the question text to a single line

Reply:
Hmm. The spec [here](https://docs.google.com/document/d/19FqLRCWQcXkMIgsR5-Isu4tHHTGx5El2cXKuEwh882I/edit#heading=h.j1x6lwgn57a) do expect us to ellipsize it at end and fit it in a single line.

Reply:
Hmm yea I think that's my takeaway from reading the spec as well. I just want to be sure because that does sound fairly limiting to me -- @ctsims Is it intended that the _question text_ (as well as the answer) should be limited to 1 line only? The reason I'm asking is because if the screen width is already being split in half between the question text and answer text, it seems like _very_ little text would fit on 1 line.

Reply:
I think it's fairly important for the utility of the widget that the app compacts things. 1 line is very little text, but generally speaking we'd expect people to be using a label above to contextualize the entry points, like

Current Stock Levels:
AZT:                     ____
Combinex:           ____
...

so I think it's fine to set this limitation for now.

Reply:
Makes sense, thanks for the explanation.